script/semantic-conventions/templates/AttributeValues.php.j2

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
should we perhaps add this to the `.gitignore` file, or are we purposefully vendoring these into a commit?

Reply:
This is a template for the [code generation](https://github.com/open-telemetry/build-tools/tree/main/semantic-conventions#code-generator). It will be used every time the semantic conventions are updated in [spec repo](https://github.com/open-telemetry/opentelemetry-specification/tree/main/semantic_conventions) to generate an updated PHP code.

A workflow will be like this:
- checkout opentelemetry-php repo
- install composer dev requirements
- make semconv
- commit and push an updated generated files

Reply:
Figured as much, just wanted to verify.  We should probably add that to the README.md as well.  Would you like to include that in this PR or a subsequent PR?

Reply:
Done. I've added some lines in README.md.